src/MinFraud/Model/ShippingAddress.php

```diff
@@ -7,11 +7,10 @@
  *
  * @property-read int|null $distanceToBillingAddress The distance in kilometers
  * from the shipping address to billing address.
- * @property-read bool|null $isHighRisk This property is true if the shipping
- * address is in the IP country. The property is false when the address is not
- * in the IP country. If the shipping address could not be parsed or was not
- * provided or the IP address could not be geolocated, then the property is
- * null.
+ * @property-read bool|null $isHighRisk This field is true if the shipping
+ * address is an address associated with fraudulent transactions. The field is
+ * false when the address is not associated with increased risk. The key will
+ * only be present when a shipping address is provided.
```

Comment:
"property" is probably better than "field" here, and the key bit is confusing given that it isn't really applicable. Maybe "The property will be `null` when a shipping address is not provided." or something.